docs/source/get-started/package-managers.rst

```diff
@@ -6,11 +6,35 @@ Use your favorite package manager to install Kokkos.
 System package managers
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-DNF
----
+Fedora
+------
+
+You may use the Fedora Project package manager to install Kokkos:
+
+.. code-block::
+
+    dnf install kokkos-devel
+
+
+Debian/Ubuntu
+-------------
+
+You may use the Debian/Ubuntu Project package manager to install Kokkos:
+
+.. code-block::
+
+    apt-get install kokkos-dev
```

Comment:
Should we refer to https://packages.debian.org/source/sid/kokkos ?
Same question here, do we know how Kokkos is being configured?  Does it enable OpenMP?
Also it looks like it should be libkokkos-dev

Reply:
See https://salsa.debian.org/debian/kokkos/-/blob/master/debian/rules?ref_type=heads#L17

Reply:
And yes, `libkokkos-dev`.

Reply:
@cedricchevalier19 true, but it should get unbundled, sometime..